Channels/Integration/01_ManageConnections.md

```diff
@@ -0,0 +1,148 @@
+[!!User interface Connections](../UserInterface/to_be_completed)
+
+# Manage the connections
+
+Establish connections via drivers to connect your marketplaces with the *Actindo Core1 Platform*. A marketplace can be either an online shop or a point of sale. You can establish an unlimited number of connections per driver.
+
```

Comment:
"You can establish an unlimited number of connections per driver." 
Stimmt/Macht das so Sinn? In Fulfillment ist es so gedacht, laut Stefan, dass man 1:1 verbindet, also 1 Driver - 1 Connection - 1 Fufiller.